Entities/Arm.h

```diff
@@ -116,7 +116,7 @@ ClassInfoGetters
 // Arguments:       None.
 // Return value:    Vector with the current absolute scene hand position.
 
-    Vector GetHandPos() const { return m_Pos + m_HandOffset; }
+    Vector GetHandPos() const { return m_JointPos + m_HandOffset; }
```

Comment:
Why did you change how get and set hand pos works? I guess this way probably makes more sense, but will this affect the various places it's used? Also, will it mess up lua scripts that use it?

Reply:
both of these are just a formality, for one reason:
during Arm's update the hand offset is moved to a position that is relative to the joint position.
and that's the way it's treated everywhere, even in the DrawHand function, as relative to the joint pos.
even in the modding guide i downloaded from the discord :).
so presumably, this fixes all the lua scripts that were using these commands until now.